apisix/plugins/kafka-proxy.lua

```diff
@@ -0,0 +1,78 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+local core = require("apisix.core")
+
+
+local schema = {
+    type = "object",
+    properties = {
+        enable_tls = {
```

Comment:
Using upstream with the presence of `tls` as `enable_tls` and `tls.verify` as `ssl_verify` will cause misunderstandings for the other standard upstream (HTTPS upstream does not support certificate verification), and the `tls` object does not have explicit "enable TLS handshake" semantics, I'm not sure we want to do that. 🤔

Reply:
The `tls` is an upstream's property, so we should configure it on the upstream. We can't use it with HTTPS now because the HTTPS requires a different port but it's not the same with kafka.

Reply:
removed, we will use upstream's tls